runtime/doc/lsp.txt

```diff
@@ -1629,7 +1632,7 @@ apply_text_edits({text_edits}, {bufnr}, {offset_encoding})
     Parameters: ~
       • {text_edits}       (table) list of `TextEdit` objects
       • {bufnr}            (integer) Buffer id
-      • {offset_encoding}  (string) utf-8|utf-16|utf-32
+      • {offset_encoding}  (string|nil) utf-8|utf-16|utf-32|
```

Comment:
I think we should rather adapt the call-sites to provide the encoding instead of adding a `nil` case, otherwise the functions won't be working correctly.

Reply:
right.  I will do it later.